test/integration/spec/services/user.js

```diff
@@ -179,10 +193,27 @@ describe('Services', function() {
       });
     });
 
-    skipInBrowser(describe)('#activate() @atlas', function() {
+    describe('#setPassword()', function() {
+      it('sets the password', function() {
+        return spark.user.setPassword({userId: party.spock.id, password: 'P@ssword123'})
+          .then(function() {
+            spark.user.verify({email: party.spock.email});
+          })
+          .then(function(res) {
+            assert.property(res, 'hasPassword');
+            assert.property(res, 'verificationEmailTriggered');
+            assert.property(res, 'sso');
+            assert.isTrue(res.hasPassword);
+            assert.isFalse(res.verificationEmailTriggered);
+            assert.isFalse(res.sso);
+          });
+      });
+    });
+
+    skipInBrowser(describe)('#activate() @CI', function() {
       createUnauthedSpark();
 
-      it('registers mobile user with a valid email address and activates account @atlas', function() {
+      xit('retrieves a valid user token using verificationToken', function() {
```

Comment:
same here, cannot test until server has endpoint that gives back verification token